docs/index.rst

```diff
@@ -18,10 +18,8 @@ Here's an example of what |docx| can do:
 |img|   ::
 
           from docx import Document
-          from docx.shared import Inches
```

Comment:
Even though it works without it (I'm not sure how?), we use Inches in the example. I'd suggest we leave it there, because "explicit is better than implicit".